tests/switchmap_/dashboard/net/routes/pages/test_devices.py

```
@@ -0,0 +1,201 @@
+else:
+    print(
+        """This script is not installed in the "{0}" directory. Please fix.\
+""".format(
+            _EXPECTED
+        )
+    )
+    sys.exit(2)
+
+# Import the switchmap classes to test
+from switchmap.dashboard.net.routes.pages import devices
+from switchmap.dashboard import uri
```

Ruff (0.8.2)

`switchmap.dashboard.uri` imported but unused

Remove unused import: `switchmap.dashboard.uri`

(F401)